README.md

```diff
@@ -124,6 +124,9 @@ app.get("/api/protected",
 );
 ```
 The scope parameter defines the required scopes, and the audience parameter is the resource URI.
+Bare in mind, while audience is optional, it is highly recommended to have some value in it.
```

Comment:
change to :
Audience is optional, it is recommended to pass the application clientId or the resource URI.

Reply:
changed